SharperUniverse/Core/ISharperSystem.cs

```diff
@@ -36,6 +36,6 @@ public interface ISharperSystem
 
 	    Task<bool> EntityHasComponentOfManagingTypeAsync(SharperEntity entity);
```

Comment:
This has yet to be updated to be synchronous...? Its only calling a synchronous method so I mean :D